.github/actions/secure-download-artifact/action.yml

```diff
@@ -26,20 +26,20 @@ runs:
       id: compute
       uses: slsa-framework/slsa-github-generator/.github/actions/compute-sha256@ab345b0851aceba69a2ce8f3d2084f6e7d887850
       with:
-        path: "${{ inputs.name }}"
+        path: "${{ inputs.path }}"
```

Comment:
IIUC, `name` is the artifact name, whereas `path` would be the path to the downloaded artifact on disk. The `compute-sha256` action would need the full path to find the file so that it can compute the sha256 for it.

I think you're right though, path needs to be provided to `download-artifact` above.
https://github.com/slsa-framework/slsa-github-generator/pull/520/files#diff-a3096b5b470f7e023efbb282e88d5d00a90dd29e183fbd1b16d6b6847ca9d3c9R22-R23

Reply:
`download-artifact` does not need a path by default: it will download the artifact by its name and store on disk the files stored for the artifact name.